pkg/engine/datasource/kafka_datasource/kafka_consumer_group_test.go

```diff
@@ -147,9 +140,8 @@ func TestKafkaMockBroker(t *testing.T) {
 
 	// Stop after 15 seconds and return an error.
 	resolveCtx := resolve.NewContext(context.Background())
-	defer resolveCtx.Context().Done()
 
-	ctx, cancel := context.WithTimeout(resolveCtx.Context(), 15*time.Second)
+	ctx, cancel := context.WithTimeout(resolveCtx.Context(), time.Hour)
```

Comment:
I think this can remain  as `15 seconds`

Reply:
yup that was left over from debugging lol 😆